frontend-react/src/contexts/SessionStorageTools.ts

```diff
@@ -0,0 +1,77 @@
+/* I put this in the `/contexts` directory because it felt
+ * right, but by no means is it a context. Feel free to move
+ * it if you can think of a better spot. */
+import { PERMISSIONS } from "../resources/PermissionsResource";
+import { groupToOrg } from "../webreceiver-utils";
+import { SessionStore } from "../hooks/UseSessionStorage";
+
+export enum GLOBAL_STORAGE_KEYS {
+    GLOBAL_BASE = "global-context-",
+    GLOBAL_ORG = "global-context-org",
+    SENDER_MODE = "global-sender-mode",
+    SENDER_NAME = "global-sender-name",
+    OKTA_ACCESS_TOKEN = "global-okta-token",
+}
+
+export function clearGlobalContext(): void {
+    for (let key in sessionStorage) {
+        sessionStorage.removeItem(key);
+    }
+}
+
+export function parseOrgs(orgs: Array<string>): Array<Partial<SessionStore>> {
+    return orgs.map((org) => {
+        if (org.includes(PERMISSIONS.SENDER)) {
```

Comment:
Maybe a comment here about case sensitivity? (string.includes is case sensitive)

Also, I hate that js has string.includes() and array.includes() and they are named the same.